### PR TITLE
#159082366 Upgrade Authors Haven Django app from Django 1.10 to Django 2.06 

### PR DESCRIPTION
**Pull request by abulojoshua1**
#### What does this PR do?
- Upgrades Authors Haven from the old Django version(1.10) to Django 2.06
#### Description of Task to be completed?
- Created a JWT authentication class
- Git ignored a .vscode/ directory
- Upgrade Authors Haven Django app version
#### How should this be manually tested?
- Clone the GitHub repo
-Cd into the application directory
- Install all dependencies using by typing 'pip install -r requirements.txt'
- Run the application on the terminal using 'python manage.py runserver'
#### Any background context you want to provide?
- Authors Haven was running on Django 1.10
#### What are the relevant pivotal tracker stories?
- #159082366

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/venv/bin/python"
+}
```
Comment (geofrocker) [inline .vscode/settings.json:3]: EOF missing here

File: authors/apps/authentication/backends.py [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+        """
+        Create a class instance here
+        """
+        pass
+
+    def authenticate(self, user_token):
+        """
+        Check Authenticity of a user token here
+        """
+        pass
```
Comment (geofrocker) [inline authors/apps/authentication/backends.py:23]: EOF missing here

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
-six==1.10.0
+astroid==2.0.1
+Django==2.0.6
+django-cors-middleware==1.3.1
+django-extensions==2.1.0
+djangorestframework==3.8.2
+djangorestframework-jwt==1.11.0
+isort==4.3.4
+lazy-object-proxy==1.3.1
+mccabe==0.6.1
```
Comment (geofrocker) [inline requirements.txt:9]: What does this do?

**Review by geofrocker — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (abulojoshua1) [reply .vscode/settings.json:3]: Thank you! I will add EOF where required. I have one question though, is it possible to add changes to the same pull request or I am required to make a new pull request?

**Pull request closed without merging**